packages/ckeditor5-remove-format/src/removeformatcommand.ts

```diff
@@ -101,9 +114,20 @@ export default class RemoveFormatCommand extends Command {
 	 * @returns The names of formatting attributes found in a given item.
 	 */
 	private* _getFormattingAttributes( item: Item | DocumentSelection, schema: Schema ) {
-		for ( const [ attributeName ] of item.getAttributes() ) {
+		for ( const [ attributeName, attributeValue ] of item.getAttributes() ) {
 			const attributeProperties = schema.getAttributeProperties( attributeName );
 
+			if ( item.is( 'element' ) && schema.isBlock( item ) && isGHSAttributeName( attributeName ) ) {
+				const {
+					styles = {},
+					classes = []
+				} = attributeValue as GHSViewAttributes;
+
+				if ( Object.keys( styles ).length || classes.length ) {
+					yield attributeName;
+				}
+			}
```

Comment:
Same as above I'd suggest using some GHS API for such checks. Example:

https://github.com/ckeditor/ckeditor5/blob/08270aafcc6e7e4fb4f4a4d2ebbc7451f031d303/packages/ckeditor5-style/src/styleutils.ts#L159-L164

Reply:
It'll not work for elements containing multiple GHS attributes, as in the comment above: https://github.com/ckeditor/ckeditor5/pull/16867#discussion_r1706475443